datafusion/proto/proto/datafusion.proto

```diff
@@ -545,8 +545,9 @@ message InListNode {
 }
 
 enum ScalarFunction {
-  Abs = 0;
-  Acos = 1;
+  //  0 was Abs
+  //  The first enum value must be zero for open enums
+  Acos = 0;
```

Comment:
This breaks the code that use 1 for Acos. I think you can keep it as it is for this PR
After moving them all to UDF, this enum can be deprecated.

I think we need to add `unknown = 0` in the future for enum.

Reply:
Good idea!
Applied the suggestions now.